src/main/java/seedu/address/logic/Messages.java

```diff
@@ -50,6 +49,8 @@ public static String format(Person person) {
                     .map(tag -> tag.toString())
                     .collect(Collectors.joining(", "));
             builder.append(tagsString);
+        } else {
+            builder.append("None");
```

Comment:
Maybe "None" can be defined as a static variable to avoid magic literals

Reply:
Good catch!